examples/benchmark2.py

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env python3
```

Comment:
This looks really nice! Users can easily make their own benchmarking schema based off this template.